app/_src/introduction/concepts.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Concepts
+---
+
+In this page we will introduce concepts that are core to understanding Kuma.
+
+## Control Plane
+
+The control-plane is the central management layer of {{ site.product_name }}. It is responsible for configuring and managing the behavior of the data plane,
+which handles the actual traffic between services.
+
+## Data-Plane Proxy / Sidecar
+
+The data-plane handles traffic between services.
+It connects to the control-plane which computes a configuration specific to it.
```

Comment:
so should it be `control plane` as in first entry on this page or `control-plane` as here?